docusign/1.0/modules/ROOT/pages/docusign-connector-studio.adoc

```diff
@@ -73,7 +73,7 @@ To add an operation for DocuSign Connector, follow these steps:
 [[configure-global-element]]
 == Configure a Global Element for the Connector
 
-When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target DocuSign system. DocuSign Connector supports Hyper connection for the Hyper configuration, and Basic authentication and PAT authentication for the Rest configuration.
+When you configure a connector, configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials that the connector requires to access the target DocuSign system. DocuSign Connector supports Authorization Connection Provider (Implicit Grant) and OAuth Connection Provider (Authorization Code Grant).
```

Comment:
I think you should add "authentication" to the text, but I don't know the product, so I could be wrong. 

DocuSign Connector supports Authorization Connection Provider (Implicit Grant) authentication and OAuth Connection Provider (Authorization Code Grant) authentication.